cmd/cerbosctl/get/internal/flagset/filter.go

```diff
@@ -15,6 +15,7 @@ type Filters struct {
 	NameRegexp      string   `help:"Filter policies by name, using regular expression"`
 	Version         []string `help:"Filter policies by version"`
 	VersionRegexp   string   `help:"Filter policies by version, using regular expression"`
+	PolicyID        []string `help:"Filter policies when listing by policy ID"`
```

Comment:
It's more natural to include the list of IDs as trailing arguments. E.g. `cerbosctl inspect policies a.b.c d.e.f`. It's also consistent with how the `get` commands already work. So, I'd mark this as an `arg` type and remove the special `policyIds` struct field in the `get` sub commands in  favour of this new filter field.